backend/api/v1beta1/go_http_client/experiment_client/experiment_service/experiment_service_get_experiment_v1_parameters.go

```diff
@@ -56,7 +56,8 @@ func NewExperimentServiceGetExperimentV1ParamsWithHTTPClient(client *http.Client
 	}
 }
 
-/*ExperimentServiceGetExperimentV1Params contains all the parameters to send to the API endpoint
+/*
```

Comment:
This file and many others modified in this PR are generated (see [this](https://github.com/kubeflow/pipelines/blob/2ad4a18efea1a9ba681259d1e739cb0fc56377fd/backend/api/v1beta1/go_http_client/experiment_client/experiment_service/experiment_service_get_experiment_v1_parameters.go#L1) for example).

IMO, it probably makes sense to add exclusions for any generated files.

Reply:
@HumairAK @mprahl do you have a preference on whether or not to exclude generated files from `go fmt`?

Reply:
@droctothorpe thanks for pointing this out. Yes, I agree that we should exclude generated files that developers are never expected to directly modify.

Reply:
Thanks for the feedback, @droctothorpe & @mprahl! All of the files modified by `go fmt ./...` were in `backend/api/v1beta1/go_http_client` and `backend/api/v2beta1/go_http_client`, which all seem to be autogenerated files. Do you still want the pre-commit hook?

Reply:
@cmdevoto how about also adding `goimports`?

Reply:
> Thanks for the feedback, @droctothorpe & @mprahl! All of the files modified by `go fmt ./...` were in `backend/api/v1beta1/go_http_client` and `backend/api/v2beta1/go_http_client`, which all seem to be autogenerated files. Do you still want the pre-commit hook?

IMO, yes. It will enforce standardized formatting for future, non-generated golang contributions.

Reply:
Version 2 of `golangci-lint` introduced [formatting](https://golangci-lint.run/usage/formatters/) with `golangci-lint fmt`(along with some other configuration file changes). Thoughts on adding `gofmt` and `goimports` to the config file and just adding another pre-commit hook to run the format command?

Reply:
@droctothorpe @mprahl Hi all! Made the changes as described in my comment above - let me know what you think